velero-plugins/common/restore.go

```diff
@@ -41,7 +41,8 @@ func (p *RestorePlugin) AppliesTo() (velero.ResourceSelector, error) {
 			"replicasets.extensions",
 			"replicationcontroller",
 			"buildconfigs",
-			"buildconfigs.build.openshift.io"},
+			"buildconfigs.build.openshift.io",
+			"horizontalpodautoscalers"},
```

Comment:
Why is this needed? I don't see anything in the below horizontalpodautoscaler restore plugin that relies on the actions of the common restore plugin here.

Reply:
I am not sure if it is needed, but on this list is all of the plugins that are present in the application.

Reply:
@ammachado This can be removed from the common plugin since we have a separate plugin registered for hpa

Reply:
@ammachado This list the resources that specifically need the actions taken in this plugin. Originally we applied these actions to all resources, but that resulted in poor performance when there were large numbers of resources being backed up and restored that didn't benefit from the actions of this plugin. Unless there's something this plugin does that your plugin relies on (and I don't think that's the case), then this is unnecessary.